src/evo/netinfo.h

```diff
@@ -203,10 +233,11 @@ class MnNetInfo final : public NetInfoInterface
         m_addr = NetInfoEntry{service};
     }
 
-    NetInfoStatus AddEntry(const std::string& service) override;
+    NetInfoStatus AddEntry(const uint8_t purpose, const std::string& service) override;
```

Comment:
enum too

Reply:
See [comment](https://github.com/dashpay/dash/pull/6666#discussion_r2286472412), resolved in latest push.